gruntfile.js

```diff
@@ -5,39 +5,41 @@
 
 'use strict';
 
+const Umberto = require( 'umberto' );
+
 module.exports = function( grunt ) {
 	require( 'load-grunt-tasks' )( grunt );
 
+	grunt.loadNpmTasks( 'grunt-contrib-connect' );
 	grunt.loadTasks( 'dev/tasks' );
-	grunt.registerTask( 'default', [ 'jsduck:docs', 'copy:source' ] );
+	grunt.registerTask( 'api', [ 'jsduck:api' ] );
+	grunt.registerTask( 'umberto', function() {
+		const done = this.async();
+
+		const skipApi = grunt.option( 'skipApi' );
+		const skipValidation = grunt.option( 'skipValidation' );
+		const dev = grunt.option( 'dev' );
+		const clean = grunt.option( 'clean' );
+
+		return Umberto.buildSingleProject( {
+			skipApi,
+			skipValidation,
+			dev,
+			clean
+		} )
+			.then( done )
+			.catch( err => {
+				grunt.log.error( `Building Documentation failed: ${ err }`);
+				done();
+			} );
+	} );
+	grunt.registerTask('docs', [ 'api', 'umberto', 'connect' ] );
```

Comment:
Default task should not use connect. It won't be useful for envs such as CI, etc.

Instead let's add an additional task that will build and set the server.

While doing that change please update building part in the readme.